### PR TITLE
changed the color of navbar and font of the title

### LINKED ISSUE #73: 🚀 [New Feature_WebApp] - Add design to the title and enhance the navbar by adding better color and other properties.
### Description of the enhancement. ✍🏻

Hi @praveenscience , I am Arun.I am planning to work on the production page which currently has some documentation.I would like to first design the navbar and the title with those fonts and colors you have used in ur landing page.

### Describe briefly how you would implement this enhancement.

```bash
Some steps to implement this enhancement-
1. Add a better font and color to the title
2. Enhance the navbar section.
```

### Screenshots Example (if applicable)

_No response_

### Additional information

_No response_

### Contributor terms

- [X] I agree to follow the Contributing Instructions

### PR DESCRIPTION
**Pull request by Arunjagan12**
## Related Issue(s)

CLoses #73 

## Proposed Changes
The color of the navbar and font of the production page's title has been changed.

## Type of Change

- [x] Enhancement

## Your Idea for Level (GSSoC)

- [ ] Level 1
- [ ] Level 2
- [ ] Level 3

## Screenshots (if applicable)

Screenshot (54)

Screenshot (55)

### CODE REVIEW

**Comment by praveenscience** [comment]
Bro, do you really think this looks good? 🤔

File: src/components/App.js [hunk 1]
```diff
@@ -18,13 +18,13 @@ class App extends Component {
     return (
       <div className="App">
         <Header
-          to="/"
```
Comment (praveenscience) [inline src/components/App.js:21]: This is a needed one, why did you remove this? 🤔

File: src/components/App.js [hunk 2]
```diff
@@ -18,13 +18,13 @@ class App extends Component {
...
+          
           Link={Link}
           dark={true}
           items={NavItems}
           className="AppHeader"
         >
-          GitHub ReadMe
```
Comment (praveenscience) [inline src/components/App.js:27]: Use prettier.

File: src/components/Docs.js [hunk 3]
```diff
@@ -20,7 +20,7 @@ class Docs extends Component {
       <div className="Docs">
         <ContainerRow fluid={true}>
           <div className="col-12">
-            <h1 className="display-4 AppHeading text-center">
+            <h1 className="display-4 AppHeading text-center font_shadows_into_light">
```
Comment (praveenscience) [inline src/components/Docs.js:23]: Can we have `-` instead of `_`?

File: src/components/Home.js [hunk 4]
```diff
@@ -7,8 +7,8 @@ const Home = () => {
       <Container fluid={true}>
         <div className="row">
           <div className="col-12">
-            <h1 className="display-4 AppHeading text-center pt-2 pb-4">
-              GitHub ReadMe: Elevating Your Projects &amp; Profile to New
+            <h1 className="display-4 AppHeading text-center pt-2 pb-4 font_shadows_into_light">
```
Comment (praveenscience) [inline src/components/Home.js:10]: Can we have `-` instead of `_`?

File: src/styles/App.scss [hunk 5]
```diff
@@ -9,4 +9,7 @@
   .AppHeading {
     font-size: 1.5em;
   }
+  .font_shadows_into_light {
```
Comment (praveenscience) [inline src/styles/App.scss:12]: Can we have `-` instead of `_`?

**Review by praveenscience — CHANGES_REQUESTED** [review]
Hey just rethink the colours dude, nice effort, but the colour doesn't seem really good. We could pretty much stay with Dark Mode?

**Comment by Arunjagan12** [comment]
I thought this will be better.But, if u mention what to change I will change it accordingly

**Comment by praveenscience** [comment]
Let's look into other harmonious colours in the palette and and show me.

**Comment by AmanJain18** [comment]
@Arunjagan12 Stick to colour that will suit dark mode, currently these bright and blue don't LGTM.
